https://issues.redhat.com/browse/ACM-30317 (#8659)
open-cluster-management/rhacm-docs: release_notes/acm_new.adoc

- The maximum compaction level for the Thanos compactor is now `4`. This configuration allows you to create one-hour downsampled blocks. As a result, memory usage for the `observability-thanos-compact-0` pod and storage consumption in the object storage bucket increase. The Thanos compactor completes more `PUT` and `DELETE` operations to create blocks with a larger duration. See link:../observability/observability_arch.adoc#observability-arch[Observability architecture] to learn more.

- You can now configure resource requirements for observability by using the `AddonDeploymentConfig` resource when you enable observability through the multicluster observability add-on. For more information, see link:../observability/obs_enable_mcoa.adoc#enable-mcoa[Enabling the multicluster observability add-on] and link:../add-ons/configure_klusterlet_addons.adoc#setting-addondeploymentconfig-klusterlet-addons[Setting up the AddOnDeploymentConfig to configure klusterlet add-ons].

> - Convert the `observability-metrics-allowlist` custom resource to `ScrapeConfig` and `PrometheusRule` resources by using the `allowlist-migration` command-line interface. Converting to `ScrapeConfig` and `PrometheusRule` resources allows for a flexible and scalable way to manage your custom allowlist configuration for the multicluster observability add-on. See link:../observability/obs_migrate_allowlist_mcoa.adoc#migrate-allowlist-mcoa[Migrating custom allowlist] to learn more.

Reviewer: Let's change this release note a bit to focus on the benefit?

> - Convert the `observability-metrics-allowlist` custom resource to `ScrapeConfig` and `PrometheusRule` resources by using the `allowlist-migration` command-line interface. Converting to `ScrapeConfig` and `PrometheusRule` resources allows for a flexible and scalable way to manage your custom allowlist configuration for the multicluster observability add-on. See link:../observability/obs_migrate_allowlist_mcoa.adoc#migrate-allowlist-mcoa[Migrating custom allowlist] to learn more.

Author: ```suggestion
- Use the `allowlist-migration` command-line interface to convert the `observability-metrics-allowlist` custom resource to `ScrapeConfig` and `PrometheusRule` resources. Converting to `ScrapeConfig` and `PrometheusRule` resources allows for a flexible and scalable way to manage your custom allowlist configuration for the `multicluster-observability-addon`. See link:../observability/obs_migrate_allowlist_mcoa.adoc#migrate-allowlist-mcoa[Migrating custom allowlist] to learn more.
```
